src/app/pages/ledger/connect-ledger-error/connect-ledger-error.layout.tsx

```diff
@@ -0,0 +1,75 @@
+import { FiCircle } from 'react-icons/fi';
+import { Box, color, Flex, Stack } from '@stacks/ui';
+
+import { BaseDrawer } from '@app/components/drawer';
+import { Caption } from '@app/components/typography';
+import ConnectLedgerError from '@assets/images/onboarding/ledger/connect-ledger-error.png';
+import { ErrorLabel } from '@app/components/error-label';
+import { PrimaryButton } from '@app/components/primary-button';
+import { Link } from '@app/components/link';
+
+import { ConnectLedgerTitle } from '../components/connect-ledger-title';
+
+interface PossibleReasonUnableToConnectProps {
+  text: string;
+}
+function PossibleReasonUnableToConnect(props: PossibleReasonUnableToConnectProps) {
+  const { text } = props;
+
+  return (
+    <Flex>
+      <Box mr="tight" mt="3px">
+        <FiCircle fill={color('text-caption')} size="4px" />
+      </Box>
+      <Caption>{text}</Caption>
+    </Flex>
+  );
+}
+
+interface ConnectLedgerErrorLayoutProps {
+  onCancelConnectLedger(): void;
+  onTryAgain(): void;
+}
+export function ConnectLedgerErrorLayout(props: ConnectLedgerErrorLayoutProps) {
+  const { onCancelConnectLedger, onTryAgain } = props;
+
+  return (
+    <BaseDrawer title={<Box />} isShowing onClose={onCancelConnectLedger}>
```

Comment:
if we need nothing here `null` or `<></>` would make more sense

Reply:
I tried that and it doesn't work with `SpaceBetween` to space the close icon on the far right.